documentation/src/docs/deployment/run-local.md

```diff
@@ -60,23 +60,61 @@ docker run -it -e HOST_IP=<ip> -p 9090:9090 -p 12345:12345 pravega/pravega:lates
 
 ## Docker Compose (Distributed Mode)
 
-Unlike other options for running locally, the docker compose option runs a full deployment of Pravega
+Unlike other options for running locally, the Docker Compose option runs a full deployment of Pravega
 in distributed mode. It contains containers for running Zookeeper, Bookkeeper and HDFS. Hence Pravega operates as if it would in production. This is the easiest way to get started with the standalone option but requires additional resources.
 
-**Prerequisite:** Docker `1.12` or later.
+1. Before attempting to deploy Pravega using this option, be sure your host machine meets the following prerequisites:
 
-Download the [docker-compose.yml](https://github.com/pravega/pravega/tree/master/docker/compose/docker-compose.yml) from github. For example:
+   * It has Docker `1.12` or later installed.
+   * It has Docker Compose installed.
+   * Download the [docker-compose.yml](https://github.com/pravega/pravega/tree/master/docker/compose/docker-compose.yml) from github. For example:
 
```

Comment:
@ravisharda canu rephrase as follows:
The following are prerequisites for deploying Pravega:

Docker 1.12 or later installed.
Docker Compose installed.
Download the docker-compose.yml(.....).

Reply:
I wanted to keep the sentences in active voice and direct. See if you like to modified sentences.

Reply:
I wanted to keep sentences in active voice and direct. Have modified them. Please check if they look OK now.

Reply:
@ravisharda GitHub

Reply:
Fixed. Thanks!